lib/inc/cpp-pcp-client/protocol/errors.hpp

```diff
@@ -0,0 +1,12 @@
+/**
+ * This is a shim to maintain compatibility with pxp-agent during development.
+ */
+#pragma once
+
+#include <cpp-pcp-client/protocol/v1/errors.hpp>
+
+namespace PCPClient {
+
+using namespace v1;
```

Comment:
Yes, I used it specifically with things like connector/connector.hpp so pxp-agent would continue to build after this was merged (and before the corresponding pxp-agent change was merged). These changes are just to add other wrappers for things that weren't used, so we don't break the API.